cypress/e2e/externalLinks/english-external-links-validator.cy.js

```diff
@@ -21,6 +21,8 @@ const excludedlinks = [
   'https://vrems.scvotes.sc.gov/Voter/Login?ref=voteusa_en',
   'https://scvotes.gov/voters/register-to-vote/?ref=voteusa_en',
   'https://info.scvotes.sc.gov/eng/ovr/start.aspx?ref=voteusa_en',
+  'https://www.vivote.gov/voters/register-to-vote/?ref=voteusa_en',
```

Comment:
can you move the new two links above line 20 to keep the sc links group separate.